### ISSUE TITLE
Poetry install fails

### ISSUE THREAD

**Issue by davisuga** [programandonocosmos/project-L-player#3]
```py

  RuntimeError

  The Poetry configuration is invalid:
    - Additional properties are not allowed ('group' was unexpected)
  

  at ~/.local/lib/python3.8/site-packages/poetry/core/factory.py:43 in create_poetry
       39│             message = ""
       40│             for error in check_result["errors"]:
       41│                 message += "  - {}\n".format(error)
       42│ 
    →  43│             raise RuntimeError("The Poetry configuration is invalid:\n" + message)
       44│ 
       45│         # Load package
       46│         name = local_config["name"]
       ```

**Comment by cl3t0** [comment]
Already fixed in #4, right?

**Comment by cl3t0** [comment]
This can be solved by updating the poetry version to something newer than 1.3.1.
